src/integrationTest/resources/Tests/CourthousesTest/courthousesPostEndpoint/requestBodySwansea.json

```diff
@@ -0,0 +1,4 @@
+{
+  "courthouse_name" : "SWANSEA",
+  "code" : "457"
```

Comment:
Wondering what is to change - should this be a Number or String for your Swansea tests?